java-frontend/src/main/java/org/sonar/java/resolve/Resolve.java

```diff
@@ -582,6 +584,10 @@ private boolean isArgumentsAcceptable(List<JavaType> argTypes, List<JavaType> fo
 
   private boolean isAcceptableType(JavaType arg, JavaType formal, boolean autoboxing) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this method 6, down to the maximum allowed 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1142)